src/cljs/org/broadinstitute/firecloud_ui/utils.cljs

```diff
@@ -40,6 +40,11 @@
   (js->clj (js/JSON.parse x)))
 
 
+(defn keywordize-keys [m]
+      (into {} (map (fn [[k v]]
+                        [(keyword k) (if (map? v) (keywordize-keys v) v)])
+                    m)))
+
```

Comment:
I should have added ... this code snippet was shamelessly stolen from @MattPutnam